src/main/java/Duke.java

```diff
@@ -1,3 +1,6 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
Maybe you could consider creating a UI class that is in change of outputting messages to the user?